server/src/languageServerBase.ts

```diff
@@ -247,9 +231,6 @@ export class LanguageServerBase {
                 });
             }
 
-            this._connection.console.log(`Fetching settings for workspace(s)`);
-            this.updateSettingsForAllWorkspaces();
```

Comment:
I don't see how this will work if we don't call updateSettingsForAllWorkspaces here. Does the LS protocol guarantee that we'll receive an onDidChangeConfiguration callback when we start?

Reply:
Yes, AFAIK client always calls it passing current client configuration.